types/xml-flow/tslint.json

```diff
@@ -0,0 +1,7 @@
+{
+    "extends": "dtslint/dt.json",
+    "rules": {
+        "strict-export-declare-modifiers": false,
```

Comment:
For documentation purposes and to help future contributors, it is preferable to [disable rules with comments](https://github.com/definitelytyped/definitelytyped#linter), and only when necessary.